list-issues raises attribute error

It's when the issue is being transformed to a dictionary. The `created_at` attribute is expected to be a `datetime.datetime`, but here it's a string.
```
[ERROR] 'str' object has no attribute 'isoformat'
[ERROR] Critical exception
Traceback (most recent call last):
  File "/home/slarse/Documents/github/repobee/repobee/src/_repobee/main.py", line 81, in main
    cli.dispatch_command(parsed_args, api, ext_commands)
  File "/home/slarse/Documents/github/repobee/repobee/src/_repobee/cli.py", line 334, in dispatch_command
    author=args.author,
  File "/home/slarse/Documents/github/repobee/repobee/src/_repobee/command.py", line 250, in list_issues
    for repo_name, issues in pers_issues_per_repo
  File "/home/slarse/Documents/github/repobee/repobee/src/_repobee/command.py", line 250, in <dictcomp>
    for repo_name, issues in pers_issues_per_repo
  File "/home/slarse/Documents/github/repobee/repobee/src/_repobee/command.py", line 247, in <dictcomp>
    data={issue.number: issue.to_dict() for issue in issues},
  File "/home/slarse/.local/share/virtualenvs/repobee-c20WEEPv/lib/python3.7/site-packages/repobee_plug/apimeta.py", line 137, in to_dict
    asdict["created_at"] = self.created_at.isoformat()
```

--- a/requirements/app.txt
+++ b/requirements/app.txt
@@ -4,3 +4,4 @@ pygithub
 colored
 repobee-plug==0.10.0
 python-gitlab==1.9.0
+maya>=0.6.1

--- a/setup.py
+++ b/setup.py
@@ -29,6 +29,7 @@ required = [
     "colored",
     "repobee-plug==0.10.0",
     "python-gitlab==1.9.0",
+    "maya>=0.6.1",
 ]
 
 setup(

--- a/src/_repobee/ext/gitlab.py
+++ b/src/_repobee/ext/gitlab.py
@@ -18,6 +18,7 @@ import pathlib
 import itertools
 from typing import List, Iterable, Optional, Generator, Tuple, Mapping
 
+import maya
 import daiquiri
 import gitlab
 import requests.exceptions
@@ -472,7 +473,9 @@ class GitLabAPI(plug.API):
                             title=issue.title,
                             body="<BODY UNAVAILABLE>",
                             number=issue.iid,
-                            created_at=issue.created_at,
+                            created_at=maya.parse(issue.created_at).datetime(
+                                naive=True
+                            ),
                             author=issue.author["username"],
                             implementation=issue,
                         )

--- a/tests/integration_tests/integration_tests.py
+++ b/tests/integration_tests/integration_tests.py
@@ -723,7 +723,9 @@ class TestListIssues:
                 repo_name, unmatched.title, ACTUAL_USER
             )
             for repo_name in repo_names
-        ]
+        ] + [
+            r"\[ERROR\]"
+        ]  # any kind of error is bad
 
         command = " ".join(
             [

